### PR TITLE
Changed "Class" to "Lesson"

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: Lesson-3-(Strings-&-Arrays).md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-## In the prep work this week (chapters 7 & 8), the students learned:
+# Lesson 3: Strings and Arrays
+
+## In the prep work this week (Chapters 7 & 8), the students learned:
```
Comment (CarlyLanglois) [inline Lesson-3-(Strings-&-Arrays).md]: maybe let's remove the chapter #s in case they change at any point?

File: Lesson-4-(Iteration).md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-## In the prep work this week (chapter 9), the students learned:
+# Lesson 4: Iteration
+
+## In the prep work this week (Chapter 9), the students learned:
```
Comment (CarlyLanglois) [inline Lesson-4-(Iteration).md]: remove chapter #? or replace it with chapter name?

**Review by CarlyLanglois — APPROVED** [review]
👍

File: Lesson-3-(Strings-&-Arrays).md (hunk 1 above)
Comment (gildedgardenia) [reply Lesson-3-(Strings-&-Arrays).md]: Done!

File: Lesson-4-(Iteration).md (hunk 2 above)
Comment (gildedgardenia) [reply Lesson-4-(Iteration).md]: Removed!

**Pull request merged**